test/parallel/test-http-pipeline-regr-2639.js

```diff
@@ -1,17 +1,17 @@
 'use strict';
 require('../common');
-const assert = require('assert');
 const http = require('http');
 const net = require('net');
+const Countdown = require('../common/countdown');
 
 const COUNT = 10;
 
-let received = 0;
+const countdown = new Countdown(1, () => server.close());
 
 const server = http.createServer(function(req, res) {
```

Comment:
This test would be better if it used `common.mustCall` here instead of Countdown since it's just 1 iteration. The `server.close()` doesn't need to be conditional.